src/rf/apps/workers/process.py

```diff
@@ -211,6 +212,11 @@ def validate_image(self, record):
         updated = status_updates.mark_image_valid(s3_uuid)
         if updated:
             log.info('Image validated %s', key)
+
+            log.info('Getting metadata from image %s', key)
+            bounds = validator.get_image_bounds()
+            image.set_bounds(bounds)
```

Comment:
I was testing some code changes and an error in my reprojection logic caused an exception.